spring-batch-integration/src/main/java/org/springframework/batch/integration/config/annotation/EnableBatchIntegration.java

```diff
@@ -75,15 +84,60 @@
  *       		.build();
  * 	}
  *
+ * 	// Middleware beans omitted
+ *
  * }
  * </pre>
  *
+ * For remote partitioning, an example of a configuration class would be:
+ *
+ * <pre class="code">
+ * &#064;Configuration
+ * &#064;EnableBatchIntegration
+ * &#064;EnableBatchProcessing
+ * public class RemotePartitioningAppConfig {
+ *
+ * 	&#064;Autowired
+ * 	private RemotePartitioningMasterStepBuilderFactory masterStepBuilderFactory;
+ *
+ * 	&#064;Autowired
+ * 	private RemotePartitioningWorkerStepBuilderFactory workerStepBuilderFactory;
+ *
+ * 	&#064;Bean
+ * 	public Step masterStep() {
+ *       	 return this.masterStepBuilderFactory
+ *       		.get("masterStep")
+ *       		.partitioner("workerStep", partitioner())
+ *       		.gridSize(GRID_SIZE)
+ *       		.outputChannel(outgoingRequestsToWorkers())
+ *       		.inputChannel(incomingRepliesFromWorkers())
+ *       		.build();
+ * 	}
+ *
+ * 	&#064;Bean
+ * 	public Step workerStep() {
+ *       	 return this.workerStepBuilderFactory
+ *       		.get("workerStep")
+ *       		.inputChannel(incomingRequestsFromMaster())
+ *       		.outputChannel(outgoingRepliesToMaster())
```

Comment:
The same `request/reply` pair is applied here as well.
This fully reflect `inbound/outbound` gateway abstraction in Spring Integration, I believe.
Or I just miss something here...